clinica/iotools/utils/data_handling_cli.py

```diff
@@ -117,31 +117,50 @@ def define_name(self):
         self._name = 'center-nifti'
 
     def define_description(self):
-        self._description = 'Center nifti of a BIDS directory'
+        self._description = 'Center NIFTI of a BIDS directory. Tool mainly used when SPM is not able \nto segment some'\
+                            + ' T1w images because the centers of these volumes are not\naligned with the origin of the'\
+                            + 'world coordinate system. By default, only\nproblematic images are converted. The rest' \
+                            + ' of the images are also copied\nto the new BIDS directory, but left untouched.'
 
     def define_options(self):
         self._args.add_argument("bids_directory",
-                                help='Path to the BIDS dataset directory in which you want to center the NIfTI files/images.')
+                                help='Path to the BIDS dataset directory in which you want to center the NIfTI '
+                                     + 'files/images.')
         self._args.add_argument("output_bids_directory",
-                                help='Path to the output directory. This is where your BIDS folder with centered nifti will appear.')
+                                help='Path to the output directory. This is where your BIDS folder with centered '
+                                     + 'nifti will appear.')
+        self._args.add_argument("--modality", '-m',
+                                help='List of modalities you want to center the NIfTI images (ex: "t1w fdg_pet dwi")'
+                                     + '. Default="t1w". All the files whose names contains one the keywords specified'
+                                     + ' in this list will be kept.',
+                                default='t1w')
+        self._args.add_argument('--center_all_files',
+                                help='Use this flag without argument to force to convert all NIFTI of the specified '
+                                     + 'modality(ies)',
+                                action='store_true',
+                                dest='center_all_files')
+        self._args.set_defaults(center_all_files=False)
```

Comment:
Shorter:

```python
        self._args.add_argument('--center_all_files', action='store_true', default=False,
                                help='Your description ending by a point.')
```
I use this e.g. in [DWI-Preprocessing CLI](https://github.com/alexandreroutier/clinica/blob/dwi_preprocessing/clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_cli.py)

Reply:
Done